src/executor/admin/GroupExecutor.cpp

```diff
@@ -14,6 +14,11 @@ namespace graph {
 folly::Future<Status> AddGroupExecutor::execute() {
     SCOPED_TIMER(&execTime_);
     auto *agNode = asNode<AddGroup>(node());
+    if (agNode->groupName() == "default") {
```

Comment:
Maybe we should check whether the group name is default in parser or validator rather than executor here.

Reply:
reasonable